OmiseSDK/Configuration.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+enum Environment {
+    case staging
+    case production
+    
+    private var vaultBaseURL: URL {
+        switch self {
+        case .staging: return URL(string: "")!
```

Comment:
How's about this? 

```suggestion
        case .staging: return URL(string: "[STAGING_URL]")!
```

Reply:
Nicely, good to read, and easy to find. 
Updated